src/pages/Favorites.js

```diff
@@ -0,0 +1,36 @@
+import { useContext } from "react";
+import Card from "../components/Card";
+import { RecipeContext } from "../context/RecipeContext";
+import styled from "styled-components";
+
+export default function Favorites() {
+  const { recipes, setRecipes } = useContext(RecipeContext);
+
+  return (
+    <>
+      <h2>Your Favorites</h2>
+      <CardsContainer>
+        {recipes
+          .filter((recipe) => recipe.favorite)
+          .map((recipe) => (
+            <Card
+              key={recipe.id}
+              id={recipe.id}
+              image={recipe.image}
+              title={recipe.title}
+              servings={recipe.servings}
+              duration={recipe.duration}
+              description={recipe.summary}
+              favorite={recipe.favorite}
+              recipes={recipes}
+              setRecipes={setRecipes}
```

Comment:
Might be a good idea to just pass the data and extract all needed data in the `Card` component?

```suggestion
              recipeData={recipe}
              recipes={recipes}
              setRecipes={setRecipes}
```